master_state.go

```diff
@@ -179,7 +180,7 @@ func newMasterStateCollector(httpClient *httpClient, slaveAttributeLabels []stri
 				for key, value := range s.Attributes {
 					normalisedLabel := normaliseLabel(key)
 					if stringInSlice(normalisedLabel, normalisedAttributeLabels) {
-						slaveAttributesExport[normalisedLabel] = value
+						slaveAttributesExport[normalisedLabel] = string(value)
```

Comment:
This will lead to double quotes being included in the value if it's a string, and ranges being represented however they are rendered in the original json. I think you should use an `interface{}` as the type and have handling for strings and floats. It's probably easiest to drop ranges for now.

Reply:
:+1: